packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,12 +10,16 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
--   [*] Image block: Replacing the media for an image set as featured prompts to update the featured image [#34666]
 
--   [**] [Gallery block] Fix crash when adding images and selecting a gallery item [#38238]
--   [***] Font size and line-height support for text-based blocks used in block-based themes [#38205]
+## 1.71.1
 
-## 1.70.0
+-   [*] Highlight text: Check if style attribute value is defined during filtering [#38670]
+
+## 1.71.0
+
+-   [*] Image block: Replacing the media for an image set as featured prompts to update the featured image [#34666]
+-   [***] Font size and line-height support for text-based blocks used in block-based themes [#38205]
+-   [*] Highlight text: Check if style attribute value is defined during filtering [#38670]
```

Comment:
Leaving a note here as a friendly reminder that when we resolve conflicts with `trunk`, this entry should be in both `1.71.1` and `1.70.3` ([reference](https://github.com/WordPress/gutenberg/pull/38721/files#r804473941)).